install/kubernetes/cilium/templates/NOTES.txt

```diff
@@ -17,6 +17,6 @@
     You have successfully installed {{ title .Chart.Name }}.
 {{- end }}
 
-Your release version is {{ .Chart.Version }}.
+Your release version is {{ .Chart.AppVersion }}.
```

Comment:
While I understand what you are trying to achieve in this PR, using AppVersion might not be a best practice as this attribute is optional ( .Chart.Version is mandatory field).

Reply:
Yeah, I saw that. I'm thinking about the situation when this repo is forked and AppVersion and Version may be different.

Reply:
It seems like you are maintaining your forked helm chart. Just curious if any of your additional features/changes can be upstream. Thanks.